### ISSUE TITLE
Latest Source Version (1.0.3) Published to PyPi Does Not Build

### ISSUE THREAD

**Issue by copperlight** [squidfunk/mkdocs-material#164]
## Description

The latest source version (1.0.3) published to PyPi does not build and install.

https://pypi.python.org/simple/mkdocs-material/

The wheel that is published for the same version does install successfully.

We have a repackaging system for distributing Python artifacts that depends on successful source builds.

### Expected behavior

A successful build and install from the source package available on PyPi.

### Actual behavior

```
Traceback (most recent call last):
  File "setup.py", line 25, in <module>
    with open("package.json") as data:
IOError: [Errno 2] No such file or directory: 'package.json'
```

### Steps to reproduce the bug

```
curl -s -o mkdocs-material-1.0.3.tar.gz 'https://pypi.python.org/packages/50/0e/5e821bb0232595e70833328bd44b8cf15fe35cdc75a262221f8248b6f5bf/mkdocs-material-1.0.3.tar.gz#md5=b14be24a0b37bc357efd6553d79e2ce0'
tar xvf mkdocs-material-1.0.3.tar.gz 
cd mkdocs-material-1.0.3
python setup.py
```

### Package versions

* Python: Python 2.7.11
* MkDocs: mkdocs, version 0.16.1
* Material: not installed

### System information

* OS: macOS Sierra 10.12.3
* Browser: n/a

**Comment by squidfunk** [comment]
I changed how the metadata for the package is obtained in the last release. Will fix it and supply a bugfix release.

**Comment by squidfunk** [comment]
Fixed with 1.0.4

**Comment by copperlight** [comment]
Nice, thanks for the quick fix.  :)
